src/components/AdjMatrix/AdjMatrixMethods.ts

```diff
@@ -8,6 +8,7 @@ import * as ProvenanceLibrary from 'provenance-lib-core/lib/src/provenance-core/
 import 'science';
 import 'reorder.js';
 import { Link, Network, Node, Cell, State } from '@/types';
+import { ScaleLinear } from 'd3';
 
```

Comment:
This is already imported on line 2
```suggestion
import { Link, Network, Node, Cell, State } from '@/types';

```